echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/HtmlToPlainText.java

```diff
@@ -0,0 +1,76 @@
+package com.netflix.spinnaker.echo.notification; /*
```

Comment:
Your copyright and import statements are a little messed up.

Reply:
damn you gradlew... what are you doing!?